packages/discovery-styles/scss/components/document-preview/_simple-document.scss

```diff
@@ -20,3 +20,33 @@
 .#{$prefix}--simple-document__content {
   max-width: 78ch;
 }
+
+$width: 500px;
```

Comment:
Why? Where did this number come from?

Reply:
It is the same width the EmptyState component uses in tooling. The message width ends up being shorter than 500px though, so its not doing anything here--but if the message changes to something longer in the future it might be helpful. I'm hesitant to add a comment stating its the width from Tooling since this is an open source component. Thoughts?

Reply:
I suggest creating a var like `$empty_state_message_width` and adding to `_vars.scss`. That way, it's top-level and not buried inside one of the components.